app/controllers/api/v1/jwt_api/response_controller.rb

```diff
@@ -37,6 +42,13 @@ def set_completed_responses
           render(status: :ok, json: [])
         end
 
+        def set_all_responses
```

Comment:
Set my responses?

Reply:
Fixed. I also renamed the other methods for index and completed to be named consistently.